src/test/scala/mesosphere/marathon/tasks/InstanceOpFactoryImplTest.scala

```diff
@@ -212,15 +210,13 @@ class InstanceOpFactoryImplTest extends UnitTest with Inside {
       val app = f.residentApp
       val volumeId = LocalVolumeId(app.id, "/path", "uuid1")
       val existingReservedInstance = f.residentReservedInstance(app.id, volumeId)
-      existingReservedInstance.agentInfo.value.host shouldBe f.defaultHostName
```

Comment:
Why was this removed?

Reply:
scheduled instances with reservations does not have tasks anymore (only when relaunching)